scripts/azure-dev/AzureDev.Config.psm1

```
@@ -104,7 +105,145 @@ function Test-AzureDevSshPublicKey {
+      $offset += [int]$fieldLength
+    }
+    if ($offset -ne $decoded.Length) {
+      return $false
+    }
+
+    $expectedFieldCount = $null
+    $expectedCurve = $null
+    $expectedPointLength = $null
+    switch ($embeddedAlgorithm) {
+      'ssh-ed25519' {
+        return $fieldLengths.Count -eq 1 -and $fieldLengths[0] -eq 32
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Test-AzureDevSshPublicKey' returns an object of type 'System.Boolean' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)